src/main/resources/reference.conf

```diff
@@ -7,4 +7,14 @@ kamon {
     # Enable of disable including tags from kamon.jaeger.environment as labels
     include-environment-tags = no
   }
+
+  modules {
+     jaeger {
+       enabled = true
+       name = "Kamon jaeger integration"
+       description = "A module description"
+       class = "kamon.jaeger.JaegerReporter"
```

Comment:
The class Member should be removed. Also, please put update the name and description. I would suggest:

name: Jaeger Reporter
description: Sends Spans to Jaeger over HTTP